src/compressor/Compressor.cc

```diff
@@ -54,6 +54,10 @@ const char *Compressor::get_comp_mode_name(int m) {
     case COMP_PASSIVE: return "passive";
     case COMP_AGGRESSIVE: return "aggressive";
     case COMP_FORCE: return "force";
+    case COMP_PASSIVE_LAZY: return "passive_lazy";
+    case COMP_AGGRESSIVE_LAZY: return "aggressive_lazy";
+    case COMP_FORCE_LAZY: return "force_lazy";
+
     default: return "???";
   }
 }
```

Comment:
The separate discussion should be if we shouldn't give different `conf.bluestore_compression_required_ratio` for scrub recompression.

Reply:
Yes, from purist's perspective we might need a completely different set of compression settings for lazy/background mode. Whether we really need that - I'm not sure...